queries/snowflake_queries/snowflake_multiple_failed_logins_followed_by_success.py

```diff
@@ -0,0 +1,10 @@
+def rule(_):
+    return True
+
+def title(event):
+    return (
+        f"Username [{event.get('user_name','<USER_NOT_FOUND>')}] from clientIP "
+        f"[{event.get('client_ip','<CLIENT_IP_NOT_FOUND>')}]] "
+        f"registered multiple failed logins followed by a success."
+    )
+ 
```

Comment:
`alert_context()` here could share how many failed logins, and the time windows involved to give investigators a stronger hint about where to look